cartographer/mapping_2d/probability_grid.h

```diff
@@ -48,6 +49,23 @@ class ProbabilityGrid {
         min_x_(limits_.cell_limits().num_x_cells - 1),
         min_y_(limits_.cell_limits().num_y_cells - 1) {}
 
+  explicit ProbabilityGrid(const proto::ProbabilityGrid& proto)
+      : limits_(proto.limits()),
+        cells_(),
+        update_indices_(proto.update_indices().begin(),
+                        proto.update_indices().end()),
+        max_x_(proto.max_x()),
+        max_y_(proto.max_y()),
+        min_x_(proto.min_x()),
+        min_y_(proto.min_y()) {
+    cells_.reserve(proto.cells_size());
+    for (auto cell : proto.cells()) {
+      // TODO(macmason): DCHECK?
```

Comment:
No DCHECKs, please. This is only for deserialization. It seems like performance shouldn't be a concern?